network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -626,10 +626,8 @@ def get_context(self, request, *args, **kwargs):
         context['categories'] = BuyersGuideProductCategory.objects.filter(hidden=False)
         context['mediaUrl'] = settings.CLOUDINARY_URL if settings.USE_CLOUDINARY else settings.MEDIA_URL
         context['use_commento'] = settings.USE_COMMENTO
-        context['pageTitle'] = f'''{pgettext(
-          'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
-          '*privacy not included'
-        )} - {self.title}'''
+        context['pageTitle'] = f'{self.title} | {gettext("Privacy & security guide")} | Mozilla Foundation'
```

Comment:
This doesn’t work unfortunately, the string is not extracted when running `inv makemessages`. I assume gettext doesn’t like the `f'` syntax, or it needs to be handled differently

Reply:
Thanks for catching this. I tried running `inv makemessages` but it didn't work for me. Do I have to run another command first to it to work? I'm getting

```bash
./translation-management.sh import
Importing latest translation files from fomo-l10n repository
cp: foundation/translations/locale/: No such file or directory
```

Reply:
note to self: follow https://github.com/mozilla/foundation.mozilla.org/blob/master/README.md#translations